cmd/webhook/main.go

```diff
@@ -148,6 +149,64 @@ func NewConfigValidationController(ctx context.Context, cmw configmap.Watcher) *
 	)
 }
 
+func NewConversionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+	var (
+		v1alpha1_ = v1alpha1.SchemeGroupVersion.Version
+		v1beta1_  = v1beta1.SchemeGroupVersion.Version
+		v1_       = v1.SchemeGroupVersion.Version
+	)
+
+	return conversion.NewConversionController(ctx,
+		// The path on which to serve the webhook
+		"/resource-conversion",
+
+		// Specify the types of custom resource definitions that should be converted
+		map[schema.GroupKind]conversion.GroupKindConversion{
+			v1.Kind("Service"): {
+				DefinitionName: "services.serving.knative.dev",
```

Comment:
I believe that this must be the GroupResource, so perhaps use that?

Reply:
done - made constants in `pkg/apis/serving `package